### PR TITLE
fix(v2): fix attributes e2e tests

### PR DESCRIPTION
**Pull request by Varixo**
# Overview

# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests / types / typos

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Use cases and why

- 1. One use case
- 2. Another use case

# Checklist:

- [ ] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: starters/e2e/e2e.attributes.spec.ts [hunk 1]
```diff
@@ -62,12 +62,12 @@ test.describe("attributes", () => {
       await stuffBtn.click();
       await expect(inputCopy).toHaveJSProperty("value", "Hello");
       await expect(inputValue).toHaveText("Hello");
-      await expect(renders).toHaveText("2");
+      await expect(renders).toHaveText("1");
```
Comment (wmertens) [inline starters/e2e/e2e.attributes.spec.ts:65]: Why the change here?

**Review by wmertens — COMMENTED** [review]
Nice!

File: starters/e2e/e2e.attributes.spec.ts (hunk 1 above)
Comment (Varixo) [reply starters/e2e/e2e.attributes.spec.ts:65]: this additional rerender was caused by removed console.log. I don't know if this is needed

**Review by wmertens — APPROVED** [review]

File: starters/apps/e2e/src/components/attributes/attributes.tsx [hunk 2]
```diff
@@ -104,9 +103,9 @@ export const AttributesChild = component$<{ v: number }>(({ v }) => {
       <div>
         {hide.value ? (
           <>
-            <label id="label" />
-            <input id="input" />
-            <svg id="svg">
+            <label id="label" key="abc" />
```
Comment (mhevery) [inline starters/apps/e2e/src/components/attributes/attributes.tsx:106]: This does not seem right. Why do we need to add the key?

**Review by mhevery — CHANGES_REQUESTED** [review]
